src/raster/r.damflood/main.c

```diff
@@ -766,76 +769,80 @@ int main(int argc, char *argv[])
             }
         }
         num_break = 0;
+
         G_message("Searching dam breach");
         for (row = 0; row < nrows; row++) {
             for (col = 0; col < ncols; col++) {
-                /*  rottura diga */
+                /*  Dam break */
                 if (m_DAMBREAK[row][col] > 0) {
                     num_break++;
                     G_message("(%d,%d)Cell Dam Breach n° %d", row, col,
                               num_break);
-                    // printf("ho trovato la rottura diga (%d,%d)=\n", row,col);
-                    // while(!getchar()){ }
+                    // printf("I found the dam break (%d,%d)=\n", row, col);
+                    // while (!getchar()) {
+                    // }
                     profondita_soglia = water_elevation -
                                         (m_z[row][col] - m_DAMBREAK[row][col]);
                     vel_0 = velocita_breccia(method, profondita_soglia);
                     volume = volume + profondita_soglia * res_ew * res_ns;
-                    // printf("Q=%.2f\n",Q);
+                    // printf("Q=%.2f\n", Q);
                     // vel_0 = 0.93 * sqrt(profondita_soglia);
                     if (vel_0 > vel_max)
                         vel_max = vel_0;
                 }
                 else {
                     G_fatal_error(
-                        _("Don't find the dambreak - Please select a correct "
-                          "map or adjust the computational region"));
+                        _("Couldn't find the dambreak. Please select a correct "
+                          "map or adjust the computational region."));
                 }
 
                 if (m_DAMBREAK[row][col] > 0) {
-                    // cambio il DTM inserendo la rottura della diga
+                    // Change the DTM by inserting the dam break
                     m_z[row][col] = m_z[row][col] - m_DAMBREAK[row][col];
                     m_lake[row][col] = 0;
                     m_h1[row][col] = profondita_soglia;
```

Comment:
What does the _profondita_soglia_ would mean. I think I understand _profondita_, similar to the French word for depth, but _soglia_?

Reply:
deepl suugest depth threshold... makes sense?

Reply:
Probably, I think I understand it now.